### PR TITLE
ocaml 5: restrict nonstd releases

### PR DESCRIPTION
**Pull request by emillon**
They rely on `Pervasives`:

    #=== ERROR while compiling nonstd.0.0.2 =======================================#
    # context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.5.0.0 | file:///home/opam/opam-repository
    # path                 ~/.opam/5.0/.opam-switch/build/nonstd.0.0.2
    # command              ~/.opam/opam-init/hooks/sandbox.sh build ocaml please.ml build
    # exit-code            2
    # env-file             ~/.opam/log/nonstd-8-d4042d.env
    # output-file          ~/.opam/log/nonstd-8-d4042d.out
    ### output ###
    # Please-> Building
    # ! cp ../nonstd.ml .
    # ! ocamlfind ocamlc  -c nonstd.ml -o nonstd.cmo
    # File "nonstd.ml", line 296, characters 22-40:
    # 296 |   let dedup ?(compare=Pervasives.compare) list =
    #                             ^^^^^^^^^^^^^^^^^^
    # Error: Unbound module Pervasives

### CODE REVIEW

File: packages/febusy/febusy.0.0.0/opam [hunk 1]
```diff
@@ -13,9 +13,9 @@ build: [
   ["jbuilder" "build" "-p" "febusy" "-j" jobs ]
 ]
 depends: [
-  "ocaml" { >= "4.03.0" }
+  "ocaml" {>= "4.03.0" & < "5.0.0"}
```
Comment (mseri) [inline packages/febusy/febusy.0.0.0/opam:16]: Is this needed? The error does not seem to be related to ocaml 5

Comment (emillon) [reply packages/febusy/febusy.0.0.0/opam:16]: Ah indeed. That'll teach me that putting unrelated changes is risky.

**Comment by kit-ty-kate** [comment]
I split off the work on `febusy` in https://github.com/ocaml/opam-repository/pull/22999

Thanks.
